go.mod

```diff
@@ -1,15 +1,15 @@
 module github.com/equinix-labs/fabric-go
 
-go 1.18
+go 1.20
```

Comment:
Is there any downside to bumping this to 1.20?  Does this force consumers to upgrade to 1.20, and if so, is that something we want to force?

Reply:
Fair point - I don't recall why I bumped this or if it happened automatically through tools and local `make` runs.

Terraform is using 1.18 today and it seems best practice is to use the minimal `go` version that works.

https://github.com/equinix/terraform-provider-equinix/blob/master/go.mod#L3

That said, I'd like to get this merged for the "May 2023" deadline. I'll follow-up with additional PRs.